frontend/src/components/PaymentConfirmationModal.tsx

```diff
@@ -66,8 +67,12 @@ function PaymentConfirmationModal({
   }, [submitStatus, paymentParams]);
 
   const onReject = async () => {
-    await new BackendClient().rejectPaymentCommand(paymentParams.referenceId);
-    setSubmitStatus("success");
+    // If on demo don't call the backend
```

Comment:
we can change it to "if (!paymentParams.demo)" and then we don't nee the "else"

Reply:
Correct. I wasn't sure if we need to take any action at all on demo. Leaving it to no action unless/until we need to do something.